labgrid/driver/commandmixin.py

```diff
@@ -34,14 +34,14 @@ def _run_check(self, cmd: str, timeout=30):
         Returns:
             List[str]: stdout of the executed command
         """
-        stdout, stderr, exitcode = self._run(cmd, timeout=timeout)
+        stdout, stderr, exitcode = self._run(cmd, timeout=timeout, codec=codec, decodeerrors=decodeerrors)
```

Comment:
Where are the variables `codec` and `decodeerrors` supposed to come from? I guess you missed updating
`def _run_check(self, cmd: str, timeout=30):` with these arguments, right?

Reply:
You should either transform the `_run()` method of `BareboxDriver`, `UBootDriver` and `SmallUBootDriver` or go without CommandMixin's convenience function `run_check` on targets with decode errors.

Reply:
You're right. I hope that these run functions will be stable soon and not have more special options.
I will add these but as unused parameters, because I don't see any use for codec's in these drivers. 
Better test functions would help a lot. In my PR for the remote_tmpdir I actually have a test case which covers the SSH driver better.